tests/unittests/OperatorTest.cpp

```diff
@@ -100,7 +89,7 @@ lessHelper(glow::PlaceholderBindings &bindings, glow::Module &mod,
 }
 
 TEST_P(OperatorTest, less_int8) {
-  ENABLED_BACKENDS(Interpreter, CPU);
+  CHECK_IF_ENABLED();
```

Comment:
Your solution uses `CHECK_IF_ENABLED` at the beginning of each test, which is  OK, but a bit intrusive. But may be it is possible to use [gtest event listeners](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#defining-event-listeners) for this purpose? IMHO, it could be a less intrusive and cleaner solution.

Reply:
I don't see a way to use an event listener for this purpose -- maybe it would work if SKIP were available (but it's not), but as it is, there's no way for the test setup to force the test to pass, only to fail.

Reply:
Yeah, it is a pity one cannot do SKIP in listeners.